linux_os/guide/system/accounts/enable_authselect/ansible/shared.yml

```diff
@@ -8,7 +8,7 @@
 
 - name: {{{ rule_title }}} - Select authselect profile
   ansible.builtin.command:
-    cmd: authselect select "{{ var_authselect_profile }}"
+    cmd: authselect select "{{ var_authselect_profile }}"; echo 1
```

Comment:
LGTM now. I will merge once you remove this modification in Ansible remediation.

Reply:
Done